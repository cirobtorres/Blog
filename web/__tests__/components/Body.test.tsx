describe("Body", () => {
  // TODO
  describe("DynamicBody", () => {
    it("renders in the document", () => {});

    it("matches the snapshot", () => {});
  });

  describe("StaticBody", () => {
    it("renders in the document", () => {});

    it("matches the snapshot", () => {});
  });
});
