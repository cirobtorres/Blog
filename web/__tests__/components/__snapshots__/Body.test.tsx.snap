// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Body DynamicBody matches the snapshot 1`] = `<DocumentFragment />`;
